sos/plugins/openstack_horizon.py

```diff
@@ -48,7 +48,8 @@ def postproc(self):
         regexp = r"((?m)^\s*(%s)\s*=\s*)(.*)" % "|".join(protect_keys)
         self.do_path_regex_sub("/etc/openstack-dashboard/.*\.json",
                                regexp, r"\1*********")
-        self.do_path_regex_sub("/etc/openstack-dashboard/local_settings",
+        self.do_path_regex_sub("/etc/openstack-dashboard/local_settings/ + \
+                               .*\.conf.*",
```

Comment:
This fixes the exception but I'm now concerned about what else may be in this directory; on the one hand, we are scooping it up as a whole directory tree, while on the other there is clearly a possibility for sensitive information here, as we have post-processing rules to remove it.

Compiled python bytecode files don't really have any place in a report so I would also be interested in a change that makes the `/etc/openstack-dashboard` data collection much more precise.